rasa/data.py

```diff
@@ -7,8 +7,11 @@
 from typing import Tuple, List, Text, Set, Union, Optional, Iterable
 from rasa.nlu.training_data import loading
 from rasa.utils.io import DEFAULT_ENCODING
+from rasa.constants import DEFAULT_E2E_TESTS_PATH
 
 logger = logging.getLogger(__name__)
+MARKDOWN_FILE_EXT = ".md"
```

Comment:
let's spell this out :-) 
```suggestion
MARKDOWN_FILE_EXTENSION = ".md"
```